internal/k8s-engine/graphql/domain/policy/converter.go

```diff
@@ -17,12 +17,26 @@ func NewConverter() *Converter {
 
 // FromGraphQLInput coverts Graphql Policy data to model.
 func (c *Converter) FromGraphQLInput(in graphql.PolicyInput) (policy.Policy, error) {
+	ifaceRules, err := c.interfaceFromGraphQLInput(in.Interface)
+	if err != nil {
+		return policy.Policy{}, err
+	}
+
+	return policy.Policy{
+		Interface: ifaceRules,
+	}, nil
+}
+
+func (c *Converter) interfaceFromGraphQLInput(in *graphql.InterfacePolicyInput) (policy.InterfacePolicy, error) {
+	if in == nil {
+		return policy.InterfacePolicy{}, nil
+	}
 	var rules policy.RulesList
 
 	for _, gqlRule := range in.Rules {
 		policyRules, err := c.policyRulesFromGraphQLInput(gqlRule.OneOf)
 		if err != nil {
-			return policy.Policy{}, errors.Wrap(err, "while getting Policy rules")
+			return policy.InterfacePolicy{}, err
```

Comment:
we should continue to use `errors.Wrap` in this file

Reply:
you're right, fixed 👍